test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] I still can see your test.md file in GitHub! When you ignore the test.md it shouldn't be displayed on GitHub! Please try to avoid this issue!